v4/feature/portfolio/src/main/java/exchange/dydx/trading/feature/portfolio/components/orders/DydxPortfolioOrdersView.kt

```diff
@@ -143,6 +152,29 @@ object DydxPortfolioOrdersView : DydxComponent {
                     PlatformDivider()
                 }
             }
+
+            item(key = "footer") {
+                CreateFooter(Modifier, state)
+            }
+        }
+    }
+
+    @Composable
+    private fun CreateFooter(modifier: Modifier, state: ViewState) {
+        if (!state.isIsolatedMarketEnabled) {
+            Box(
+                contentAlignment = Alignment.Center,
+                modifier = modifier.fillMaxWidth()
+                    .padding(vertical = 24.dp),
```

Comment:
can we reduce top padding to 12?

Reply:
Update to 16...  12 looks too tight
![Screenshot_1716587165](https://github.com/dydxprotocol/v4-native-android/assets/102453770/9c6c3a38-e395-48ac-be6c-1aa0419e8908)